src/components/directory-content.js

```diff
@@ -817,6 +818,19 @@ const DirectoryContent = () => {
         );
     }
 
+    const emptyMenu = () => {
+        return !(
+            allowsRename() ||
+            allowsPublishability() ||
```

Comment:
The code of the isAllowed() method can be moved into AllowPublishability() and isAllowed() can be removed ?

Reply:
Not really I think, because its consistent with the isAllowed() function in directory-container. Actually this code needs a refactoring when we will add more functionalities in the directory-content toolbar, and a generalized management of actions on multi-selections.
We mainly check for selected items, types of items and privileges (user and access rights) for consistent allowed actions.